tests/conftest.py

```diff
@@ -21,10 +22,25 @@ def context() -> Dict[str, str]:
     return {"project_slug": "example"}
 
 
+@pytest.fixture(params=["vanilla", "cruft"])
+def flavor(request: pytest.FixtureRequest) -> str:
+    """Test flavor, either vanilla cookiecutter or cruft."""
+    out: str = request.param  # type: ignore[attr-defined]
```

Comment:
```suggestion
    out: str = request.param
```
This `# type: ignore` can be removed by updating Pytest to >= 7.1 I believe.
I tried updating it with Poetry, but I believe it'd be better to update all dependencies at once.